packages/atomic/src/components/commerce/product-template-components/atomic-product-image/atomic-product-image.tsx

```diff
@@ -137,6 +164,21 @@ export class AtomicProductImage implements InitializableComponent {
     return Array.isArray(value) ? value : [value];
   }
 
+  private get imagesAlt() {
+    if (this.imagesAltField) {
+      const value = ProductTemplatesHelpers.getProductProperty(
+        this.product,
+        this.imagesAltField
+      );
+
+      if (Array.isArray(value)) {
+        return value.map((v) => `${v}`.trim());
```

Comment:
Why are we trimming the values only when it's an array?

Reply:
Missed that, thanks!